components/modal/stories/modal.stories.js

```diff
@@ -3,23 +3,37 @@ import { Template } from "./template";
 
 export default {
   title: "Modal",
-  description: "The Modal component is...",
+  description: "A modal component used primarily by Dialog.",
   component: "Modal",
   argTypes: {
-    size: {
-      name: "Size",
-      type: { name: "string", required: true },
+    isOpen: {
```

Comment:
Would you add a content input here so other components can leverage the Modal component and pass in their own inputs? You can hide it from the controls panel with: `content: { table: { disable: true } }`

Reply:
I've added that in [b3f186c](https://github.com/adobe/spectrum-css/pull/1655/commits/b3f186c6fcd11943b462ebee9e1ccd6e754202cf), trying to match the format of some of the other stories that are using an array.